pkg/application/app.go

```diff
@@ -50,7 +50,7 @@ func LoadFromFile(fileName string) (*Application, error) {
 	_, err = os.Stat(fileName)
 	if err != nil {
 		if os.IsNotExist(err) {
-			return newApplication(nil, tm), nil
+			return nil, err
```

Comment:
Returning `nil`: +1.
Returning `err`: -1 as it happens when the application was created before.